Content.Server/Atmos/Piping/Unary/Components/GasThermoMachineComponent.cs

```diff
@@ -93,5 +89,11 @@ public sealed class GasThermoMachineComponent : Component
         [DataField("machinePartTemperature", customTypeSerializer: typeof(PrototypeIdSerializer<MachinePartPrototype>))]
         public string MachinePartTemperature = "Capacitor";
 
+        /// <summary>
+        /// Last amount of energy added/removed from the attached pipe network
+        /// </summary>
+        [DataField("lastEnergyDelta")]
+        [ViewVariables(VVAccess.ReadWrite)]
```

Comment:
Does this need to be RW? It looks like it should be read-only since it's written to only in the ThermoMachineSystem

Reply:
RW should be default for components IMO. Sure it's not useful, but having to pick and choose is meh.